app/src/main/java/com/kickstarter/ui/activities/compose/FormularyScreen.kt

```diff
@@ -112,7 +112,7 @@ fun FormularyScreen(
             value = outputs.email().subscribeAsState(initial = "").value,
             onValueChange = {},
             readOnly = true,
-            label = { Text(stringResource(id = R.string.email)) },
```

Comment:
Can someone explain to me where this `R.string.email` string went? It's not in strings_i18n.xml